GitUI/UserControls/RevisionGrid/Graph/Lane.cs

```diff
@@ -1,14 +1,4 @@
 ﻿namespace GitUI.UserControls.RevisionGrid.Graph
 {
-    public sealed class Lane
-    {
-        public int Index;
-        public LaneSharing Sharing;
-
-        public Lane(int index, LaneSharing sharing)
-        {
-            Index = index;
-            Sharing = sharing;
-        }
-    }
+    public record Lane(int Index, LaneSharing Sharing);
```

Comment:
❓ 
```suggestion
    public readonly record struct Lane(int Index, LaneSharing Sharing);
```

Reply:
`RevisionGraphRow.MoveLanesRight()` needs to increment `.Index`. A dictionary of structs cannot be updated without removing and adding again.